lib/util/import-target.js

```diff
@@ -239,6 +245,24 @@ module.exports = class ImportTarget {
         return [this.options.basedir]
     }
 
+    /**
+     * @param {string} baseDir
+     * @param {unknown} error
+     * @returns {void}
+     */
+    handleResolutionError(baseDir, error) {
+        if (error instanceof Error === false) {
+            throw error
+        }
+
+        // This is the usual error
+        if (error.message === `Can't resolve '${this.name}' in '${baseDir}'`) {
+            return
+        }
```

Comment:
I'm a bit worried that this check is error-prone: the error message may change in a later release.

Reply:
This is fair, I wonder if the error message should just be piped straight through :thinking:

Reply:
👍 from me!